ladybug/wea.py

```diff
@@ -57,6 +77,24 @@ def header(self):
             "site_elevation %.1f\n" % self.location.elevation + \
             "weather_data_file_units %d\n" % self.timestep
 
+    def to_json(self):
+        """Write Wea to json file
+            {
+            "location": {} , // ladybug location schema
+            "direct_normal_radiation": (), // Tuple of hourly direct normal
+                radiation
+            "diffuse_horizontal_radiation": (), // Tuple of hourly diffuse
+                horizontal radiation
+            "timestep": float //timestep between measurements, default is 1
+            }
+        """
+        return {
+                "location": self.location.to_json(),
+                "direct_normal_radiation": self.direct_normal_radiation,
```

Comment:
We need to also jsonify `direct_normal_radiation` and `diffuse_horizontal_radiation` which are from type [DataCollection](https://github.com/ladybug-tools/ladybug/blob/ebe82c3f0a9cbe424a95c0a3ac99a728010b26c1/ladybug/datacollection.py#L10).

Reply:
Not sure how to go about jsonifying DataCollection objects... Would it be:
'''python
{
    "header": {
        "location": "", // Location
        "data_type": "", // Datatype
        "unit": "", // Unit
        "analysis_period": tuple // Analysis period tuple eg: ((1,1,1),(24,31,12))
        },
    "datapoint": [] // list of datapoints
}
'''